engine/engine.go

```diff
@@ -112,18 +112,18 @@ func New(c *Config, hs map[string]HandlerFunc) (*Engine, error) {
 		return nil, err
 	}
 
+	consumer, err := ssqs.New(&ssqs.Queue{
+		URL:               c.ConsumerQueueURL,
+		VisibilityTimeout: int64((time.Minute * 30).Seconds()),
+	})
+
 	e := &Engine{
 		config:    c,
 		keyring:   k,
 		handlers:  hs,
 		semaphore: make(chan struct{}, maxConcurrentHandlers),
 		producer:  sqs.New(a, aws.Regions[c.Region]),
-		consumer: ssqs.New(&ssqs.Queue{
-			Name:              c.ConsumerQueue,
```

Comment:
If this config isn't being used anywhere else then it should be removed from the config package as well.